### PR TITLE
TH2-4379 BufferedJSONProcessor

### PR DESCRIPTION
**Pull request by 57R1D3R**
CHANGE: v5/GetEvents, v5/GetMessages, v6/GetMessages switched to BufferedJSONProcessor

### CODE REVIEW

File: th2_data_services/provider/v5/commands/http.py [hunk 1]
```diff
@@ -674,11 +677,10 @@ def __init__(
         self._char_enc = char_enc
         self._decode_error_handler = decode_error_handler
         self._cache = cache
+        self.sse_handler = sse_handler or get_default_sse_adapter()
 
     def handle(self, data_source: HTTPProvider5DataSource) -> Data:  # noqa: D102
-        source = partial(self.__handle_stream, data_source)
-        adapter = SSEAdapter()
-        return Data(source).map(adapter.handle).use_cache(self._cache)
+        source = partial(self.sse_handler.handle, partial(self.__handle_stream, data_source))
```
Comment (ConnectDIY) [inline th2_data_services/provider/v5/commands/http.py:683]: Forgot to return Data object

File: th2_data_services/utils/json.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    def _decode_with_buffer(self, x: str):
+        if len(self.buffer) < self.buffer_limit:
+            self.buffer.append(x)
+        else:
+            yield from self.from_buffer()
+            self.buffer = [x]
+
+    def fin(self):
+        if self.buffer:
+            yield from self.from_buffer()
```
Comment (ConnectDIY) [inline th2_data_services/utils/json.py:34]: Docstrings required

**Review by ConnectDIY — CHANGES_REQUESTED** [review]

**Comment by ConnectDIY** [comment]
@xHacka add also release notes, please

**Review by xHacka — COMMENTED** [review]
Branch updated

**Review by ConnectDIY — APPROVED** [review]

**Pull request merged**